pkg/operator/options/options.go

```diff
@@ -90,7 +90,7 @@ func (o *Options) AddFlags(fs *FlagSet) {
 	fs.IntVar(&o.KubeClientQPS, "kube-client-qps", env.WithDefaultInt("KUBE_CLIENT_QPS", 200), "The smoothed rate of qps to kube-apiserver")
 	fs.IntVar(&o.KubeClientBurst, "kube-client-burst", env.WithDefaultInt("KUBE_CLIENT_BURST", 300), "The maximum allowed burst of queries to the kube-apiserver")
 	fs.BoolVarWithEnv(&o.EnableProfiling, "enable-profiling", "ENABLE_PROFILING", false, "Enable the profiling on the metric endpoint")
-	fs.BoolVarWithEnv(&o.EnableLeaderElection, "leader-elect", "LEADER_ELECT", true, "Start leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.")
+	fs.BoolVarWithEnv(&o.DisableLeaderElection, "disable-leader-elect", "DISABLE_LEADER_ELECT", false, "Disable the leader election client before executing the main loop. Disable when running replicated components for high availability is not desired.")
```

Comment:
Can we match the name of the environment variable and the CLI arg with the option name. e.g. `DISABLE_LEADER_ELECTION` and `--disable-leader-election`

Reply:
Done.